tests/e2e/users/create-meta-db.wdio-spec.js

```diff
@@ -0,0 +1,43 @@
+const _ = require('lodash');
+const utils = require('../../utils');
+const usersPage = require('../../page-objects/users/users.wdio.page');
+const commonElements = require('../../page-objects/common/common.wdio.page');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const addUserModal = require('../../page-objects/users/add-user-modal.wdio.page');
+
+const username = 'fulltester';
+const fullName = 'Roger Milla';
+const password = 'StrongP@ssword1';
+
+const options = {
+  auth: { username: username, password },
+  method: 'GET',
+  userName: username
+};
+
+describe('Create user meta db : ', () => {
+
+  before(async () => await loginPage.cookieLogin());
+  after(async () => await utils.deleteUsers([{ username: username }], true));
```

Comment:
I don't think this is required, we already have a global `after` hook that deletes users: https://github.com/medic/cht-core/blob/master/tests/wdio.conf.js#L311 . 
Maybe we should change that one to always call `deleteUsers` passing `true` as the second param.